samples/subsys/usb_c/source/src/main.c

```diff
@@ -15,8 +15,8 @@ LOG_MODULE_REGISTER(main, LOG_LEVEL_DBG);
 
 #include "power_ctrl.h"
 
-#define PORT1_NODE		DT_NODELABEL(port1)
-#define PORT1_POWER_ROLE	DT_ENUM_IDX(DT_NODELABEL(port1), power_role)
+#define PORT1_NODE		DT_ALIAS(usbc_port0)
```

Comment:
Update this with the same naming convention that you decide on for the other sample.  `USBC_PORT0_NODE` or `USBC_PORT_NODE`.